### DISCUSSION TITLE
Chown command does not work

### DISCUSSION THREAD

**Discussion by Nemachtilli** [ohmyzsh/ohmyzsh#10042] [Help] [answered]
in writing sudo chown -R $USER:root /var/www

chown: invalid user: "usernameoot"

if I stop using oh my zsh and enter the command in the normal bash It works well

try to change my locale

LANG=es_MX.utf8
LANGUAGE=es_MX:es
LC_CTYPE="es_MX.utf8"
LC_NUMERIC="es_MX.utf8"
LC_TIME="es_MX.utf8"
LC_COLLATE="es_MX.utf8"
LC_MONETARY="es_MX.utf8"
LC_MESSAGES="es_MX.utf8"
LC_PAPER="es_MX.utf8"
LC_NAME="es_MX.utf8"
LC_ADDRESS="es_MX.utf8"
LC_TELEPHONE="es_MX.utf8"
LC_MEASUREMENT="es_MX.utf8"
LC_IDENTIFICATION="es_MX.utf8"
LC_ALL=

What is the error?

**Comment by rwmitchell** [comment]
When in zsh and OMZ, what is the value of $USER ?

**Comment by rwmitchell** [comment, marked as answer]
see: https://zsh.sourceforge.io/Doc/Release/Expansion.html

the colon is a modifier in zsh, this has nothing to do with OMZ.

You can type your command like this:
```echo $USER":root"```
to get the results you're after.
